include/kllvm/ast/AST.h

```diff
@@ -963,7 +966,17 @@ class kore_definition {
    *
    *   S |-> {T . S is a subsort of T}
    */
-  [[nodiscard]] SubsortMap get_subsorts() const;
+  [[nodiscard]] SubsortMap get_subsorts();
+
+  /*
+   * Build this definition's inverted subsort relation from axioms that have the
```

Comment:
My point here was that "supersort" captures precisely what you mean by "inverted subsort"; the sub- and supersort relations are indeed inverses of one another, but it's good to keep the naming consistent.